presto-product-tests/src/main/java/com/facebook/presto/tests/AlterTableTests.java

```diff
@@ -33,67 +32,81 @@
 public class AlterTableTests
         extends ProductTest
 {
-    private static final String TABLE_NAME = "table_name";
-    private static final String RENAMED_TABLE_NAME = "renamed_table_name";
-
-    @BeforeTestWithContext
-    @AfterTestWithContext
-    public void dropTestTables()
-    {
-        try {
-            query(format("DROP TABLE IF EXISTS %s", TABLE_NAME));
-            query(format("DROP TABLE IF EXISTS %s", RENAMED_TABLE_NAME));
-        }
-        catch (Exception e) {
-            Logger.get(getClass()).warn(e, "failed to drop table");
-        }
-    }
-
     @Test(groups = {ALTER_TABLE, SMOKE})
     public void renameTable()
     {
-        query(format("CREATE TABLE %s AS SELECT * FROM nation", TABLE_NAME));
+        String tableName = Tables.randomTableName();
```

Comment:
Rename to uniqueTableName() and use an AtomicInteger underneath to avoid test non-determinism / flakiness.

Same as randomTalbeName in AlterTableTests. Also, deduplicate the two methods (use Tables.uniqueTableName in AlterTableTests)